src/search/sr_optimizers.c

```
@@ -330,3 +425,64 @@ void sr_optimizer_print_help(FILE *output)
     fprintf(output, "\n");
   }
 }
+
+int sr_parse_int_arg(const char *val, int *out)
+{
+  char *end = NULL;
```

Clang (14.0.6)

[warning] variable 'end' is not initialized

(cppcoreguidelines-init-variables)